pkg/proxy/iptables/proxier.go

```diff
@@ -155,6 +153,66 @@ type serviceInfo struct {
 	serviceLBChainName       utiliptables.Chain
 }
 
+func (info *serviceInfo) ClusterIP() string {
```

Comment:
@m1093782566  suggest to add some instructions of the funcs.

Reply:
Sure, will do that.